docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md

```diff
@@ -11,447 +11,184 @@ menu:
 
-You can obtain information for alert templates from YugabyteDB Anywhere by executing the following command:
+1. Click **Add Channel** and then complete the **Create new alert channel** dialog.
 
-```sh
-aakra01@UCTDES01 ~ k exec yb-support-platform-yugaware-0 -c prometheus -n yb-platform -- ls /opt/yugabyte/prometheus/rules |more -5 yugaware.ad.034b2eec-23c6-4ef2-8a25-aab9820a4086.yml yugaware.ad.04b30133-e62d-44ef-a7c1-7aadb6bf4c39.yml yugaware.ad.0d28c7a4-e44f-4247-8343-029662075233.yml yugaware.ad.155697f8-f928-40a3-98e5-78f0b645c4f6.yml yugaware.ad.18175790-3fae-4008-83a8-41bb221a2cf7.yml --More--   aakra01@UCTDES01 ~ k get configmap/yb-support-platform-yugaware-prometheus-config -n yb-platform -o yaml |grep -i rule    rule_files:    - '/opt/yugabyte/prometheus/rules/yugaware.ad.*.yml'
-```
+1. Select the target for the channel.
 
--->
+    If you select **Email** as the target, perform the following:
 
-## Define notification channels
+      - Provide a descriptive name for your channel.
 
-In YugabyteDB Anywhere, a notification channel defines how an alert is issued (via an email, a Slack message, a webhook message, or a PagerDuty message) and who should receive it.<br>You can create a new channel, as well as modify or delete an existing one as follows:
+      - To use the details provided on the **Health** tab for the recipients and the mail server, select the **Use recipients specified on Health tab** and **Use SMTP server specified on Health tab** options. See [Configure health check](#configure-health-check).
 
-- Navigate to **Alert Configurations > Notification Channels**, as per the following illustration:
+        Otherwise, use the **Emails** field to enter one or more valid email addresses separated by commas, and enter the Simple Mail Transfer Protocol (SMTP) settings to use.
 
-  <br>
+    If you select **Slack** as a notification delivery method, perform the following:
 
-  ![Notification channel](/images/yp/config-alerts7.png)
+      - Provide a descriptive name for your channel.
 
-  <br>
+      - Use the **Slack Webhook URL** field to enter a valid URL.
 
-- To create a new channel, click **Add Channel** and then complete the **Create new alert channel** dialog shown in the following illustration:<br><br>
+    If you select **PagerDuty** as a notification delivery method, perform the following:
 
-  ![New channel](/images/yp/config-alerts6.png)
+      - Provide a descriptive name for your channel.
 
-  <br>
+      - Enter a PagerDuty API key and service integration key.
 
-  If you select **Email** as a notification delivery method, perform the following:
+    If you select **WebHook** as a notification delivery method, perform the following:
 
-  - Provide a descriptive name for your channel.
+      - Provide a descriptive name for your channel.
 
-  - Use the **Emails** field to enter one or more valid email addresses separated by commas.
+      - Use the **Webhook URL** field to enter a valid URL.
```

Comment:
We added support for authentication config for webhook channel.
It looks like this:

![image](https://user-images.githubusercontent.com/80941592/236425629-bd349cd2-c0e2-4efc-8a58-f718b7afdc53.png)
![image](https://user-images.githubusercontent.com/80941592/236425680-68ef71b9-32df-4d97-af19-950c026156ff.png)

May want to add this info here as well.